<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetAcquireParamForm</class>
 <widget class="QWidget" name="SetAcquireParamForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="maximumSize">
      <size>
       <width>72</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageCameraParam">
      <attribute name="label">
       <string>相机参数</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pageShowParam">
      <attribute name="label">
       <string>显示参数</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="pageAcquireParam">
      <attribute name="label">
       <string>采集参数</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>toolBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
